CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- **[BREAKING CHANGE]** Add default horizontal padding in base `Item`.
```

Comment:
It reminds me a PR that I did and rollback ;)

Reply:
ahah, yep it's a dead PR. I realized after that it was much more complicated than just that.
I will decline it now.